source/backend/dal/Repositories/PropertyRepository.cs

```diff
@@ -283,17 +284,24 @@ public PimsProperty GetAllAssociationsById(long id)
         /// <summary>
         /// Update the passed property in the database assuming the user has the required claims.
         /// </summary>
-        /// <param name="property"></param>
-        /// <returns></returns>
+        /// <param name="property">The property to update.</param>
+        /// <param name="overrideLocation">Whether to update the property spatial location with the incoming value. Defaults to false.</param>
+        /// <returns>The updated property.</returns>
         public PimsProperty Update(PimsProperty property, bool overrideLocation = false)
         {
             property.ThrowIfNull(nameof(property));
 
             var propertyId = property.Internal_Id;
-            var existingProperty = this.Context.PimsProperties
+            var existingProperty = Context.PimsProperties
                 .Include(p => p.Address)
                 .FirstOrDefault(p => p.PropertyId == propertyId) ?? throw new KeyNotFoundException();
 
+            // PSP-7876 prevent editing on retired properties
```

Comment:
nit: we probably don't need this comment with the ticket number?

Reply:
We sometimes put the JIRA # to refer back to business logic that was relevant at the time. Will remove but this "retired" thing will become an integral part of the system and wanted to put that # there to reflect the business decision.